template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/zeppelin/ZeppelinS3CloudStorageRoleConfigProvider.java

```diff
@@ -64,12 +64,13 @@ protected String getCloudStorageProperty(TemplatePreparationObject source) {
 
         ConfigUtils.getStorageLocationForServiceProperty(source, ZEPPELIN_NOTEBOOK_S3_BUCKET).ifPresent(
                 storageLocation -> hmsCloudStorage.append(
-                        ConfigUtils.getSafetyValveProperty(ZEPPELIN_NOTEBOOK_S3_BUCKET, storageLocation.getValue()))
+                        ConfigUtils.getSafetyValveProperty(ZEPPELIN_NOTEBOOK_S3_BUCKET,
+                                storageLocation.getValue().replace(S3A_PREFIX, "")))
```

Comment:
See below; some validation might be useful to allow only values without any path separator (defaults to `/`) after removing the scheme prefix.

Reply:
Used REGEX to extract s3 bucket and also added unit tests for various s3 bucket scenarios.